.travis.yml

```diff
@@ -81,3 +75,12 @@ matrix:
     - go: 1.13.x
       os: windows
       env: GO111MODULE=off GOFLAGS=
+    - go: 1.14.x
+      os: linux
+      env: GO111MODULE=off GOFLAGS=
```

Comment:
I'm not sure that this is right - should it not be module=on? It doesn't make sense to test 1.13 and 1.14 with the same config